data/datacatalogue/molgenis.csv

```diff
@@ -120,14 +134,22 @@ Databanks,,multipleEntries,bool,,,,,,,,,"Can there be multiple entries for a sin
 Databanks,,updates,heading,,,,,,,,,Information on the regularity of updates and time lags
 Databanks,,refresh,int,,,,,,,,,Average number of days between refresh of data bank with new records
 Databanks,,lagTime,int,,,,,,,,,"How many days is the lag time after refresh before a record can be extracted? (e.g., a lag time may occur if the originator conducts quality checks)"
+Databanks,,preservation,bool,,,,,,,,,Are records preserved in the data bank indefinitely?
+Databanks,,preservationLength,text,,,,,,,,,"If no to the above, for how long are records preserved in the data bank?"
 Databanks,,refreshPeriod,ontology_array,,,CatalogueOntologies,Months,,,,,"If data are refreshed on fixed dates (e.g., every June and December), when are the refreshes scheduled? Select all that apply from the following:"
 Databanks,,dateLastRefresh,date,,,,,,,,,Date of last update/refresh
 Databanks,,quality,heading,,,,,,,,,List of relevant studies conducted using the data bank
 Databanks,,numberOfRecords,int,,,,,,,,,Total number of unique records captured in the data bank (most recent count)
 Databanks,,completeness,text,,,,,,,,,"Describe the completeness of the data bank (e.g., variables with more or fewer missing values)."
 Databanks,,completenessOverTime,text,,,,,,,,,"Describe any changes in completeness of the data bank (e.g., variables with more or fewer missing values) that have occurred  over time"
+Databanks,,completenessResults,text,,,,,,,,,What methods or processes are applied to check completeness of the data bank?
```

Comment:
This name is confusing if I see description. Then I would expect 'completessChecks'